lib/active_encode/engine_adapters/ffmpeg_adapter.rb

```diff
@@ -119,15 +119,15 @@ def build_input encode
         input = ActiveEncode::Input.new
         metadata = get_tech_metadata(working_path("input_metadata", encode.id))
         input.url = metadata[:url]
-        input.assign_attributes(metadata.tap { |h| h.delete(:url) })
+        input.assign_tech_metadata(metadata.reject { |k, v| k == :url })
```

Comment:
Since you're using `#assign_tech_metadata`, I don't think you need to reject the `:url` key/value pair since it operates off of a whitelist.